### PR TITLE
Bluetooth hids mouse sample testing sample changes

### PR DESCRIPTION
**Pull request by kapi-no**

### CODE REVIEW

**Comment by kapi-no** [comment]
ncs-si-muffin-doc, ncs-si-dragoon-doc teams should be replaced by the ncs-si-bluebagel-doc team.

I will apply this change in a follow-up PR

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 1]
```diff
@@ -161,6 +193,10 @@ Testing with nRF Connect for Desktop
 
 To test with `nRF Connect for Desktop`_, complete the following steps:
 
+.. note::
+   To execute the testing steps below, you must build the sample in the configuration that is compatible with `nRF Connect for Desktop`_.
+   See the :ref:`peripheral_hids_mouse_ncd_build` section to learn how build the compatible sample configuration.
```
Comment (peknis) [inline samples/bluetooth/peripheral_hids_mouse/README.rst:198]:
```suggestion
   See the :ref:`peripheral_hids_mouse_ncd_build` section to learn how to build a compatible sample configuration.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 2]
```diff
@@ -107,14 +121,32 @@ To build this sample with the :ref:`ble_rpc` library, add the following paramete
...
+   west build -b *board_target* -- -DCONFIG_BT_DIRECTED_ADVERTISING=n -DCONFIG_BT_PRIVACY=n
 
 .. note::
-   |54H_engb_2_8|
+   If you want to combine this build configuration with the :ref:`peripheral_hids_mouse_bt_rpc_build`, use the following command:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west build -b *board_target* -S nordic-bt-rpc -- -DFILE_SUFFIX=bt_rpc -DCONFIG_BT_DIRECTED_ADVERTISING=n -DCONFIG_BT_PRIVACY=n -Dipc_radio_CONFIG_BT_PRIVACY=n
```
[the remark this answers is not in the document]
Comment (alstrzebonski) [reply samples/bluetooth/peripheral_hids_mouse/README.rst:149]: I'd prefer to keep it as is. I don't like the idea of building from the testcase - it's less straight forward.

**Review by alstrzebonski — APPROVED** [review]

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 3]
```diff
@@ -97,6 +97,20 @@ The HID service specification does not require encryption (:kconfig:option:`CONF
...
 ********************
 
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_hids_mouse`
+
+.. include:: /includes/build_and_run_ns.txt
+
+.. include:: /includes/nRF54H20_erase_UICR.txt
+
+.. note::
+   |54H_engb_2_8|
```
[the remark this answers is not in the document]
Comment (kapi-no) [reply samples/bluetooth/peripheral_hids_mouse/README.rst:107]: this is maintained by the tech writers and if necessary they will probably align it for NCS 2.9

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 4]
```diff
@@ -107,14 +121,32 @@ To build this sample with the :ref:`ble_rpc` library, add the following paramete
...
-.. include:: /includes/build_and_run_ns.txt
+.. _peripheral_hids_mouse_ncd_build:
 
-.. include:: /includes/nRF54H20_erase_UICR.txt
+nRF Connect for Desktop build
+=============================
+
+To build this sample in the configuration variant that is compatible with `nRF Connect for Desktop`_, disable the following Bluetooth features:
+
+* Privacy (:kconfig:option:`CONFIG_BT_PRIVACY`) - `nRF Connect for Desktop`_ does not support Bluetooth Privacy very well and does not allow distribution of the Identity Resolving Key (IRK) during the Bluetooth pairing procedure.
```
[the remark this answers is not in the document]
Comment (kapi-no) [reply samples/bluetooth/peripheral_hids_mouse/README.rst:132]: you are right - this sounds a bit bad. I will change to "fully"

**Comment by kapi-no** [comment]
addressed review comments

File: samples/bluetooth/peripheral_hids_mouse/README.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (kapi-no) [reply samples/bluetooth/peripheral_hids_mouse/README.rst:149]: yes, this also has educational value as it demonstrates what Kconfig option in the Bluetooth stack is responsible for enabling Privacy. We have never relied on the `-T` build variant in the documentation of our projects.

File: samples/bluetooth/peripheral_hids_mouse/README.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (peknis) [reply samples/bluetooth/peripheral_hids_mouse/README.rst:107]: @annwoj

**Review by zycz — APPROVED** [review]

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 5]
```diff
@@ -107,14 +121,32 @@ To build this sample with the :ref:`ble_rpc` library, add the following paramete
...
-.. include:: /includes/build_and_run_ns.txt
+.. _peripheral_hids_mouse_ncd_build:
 
-.. include:: /includes/nRF54H20_erase_UICR.txt
+nRF Connect for Desktop build
+=============================
+
+To build this sample in the configuration variant that is compatible with `nRF Connect for Desktop`_, disable the following Bluetooth features:
+
+* Privacy (:kconfig:option:`CONFIG_BT_PRIVACY`) - `nRF Connect for Desktop`_ does not fully support the Bluetooth Privacy feature by disallowing distribution of the Identity Resolving Key (IRK) during the pairing procedure.
```
Comment (peknis) [inline samples/bluetooth/peripheral_hids_mouse/README.rst:132]:
```suggestion
* Privacy (:kconfig:option:`CONFIG_BT_PRIVACY`) - the `Bluetooth Low Energy app`_ does not fully support the Bluetooth Privacy feature by disallowing distribution of the Identity Resolving Key (IRK) during the pairing procedure.
```

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 6]
```diff
@@ -107,14 +121,32 @@ To build this sample with the :ref:`ble_rpc` library, add the following paramete
...
-.. |sample path| replace:: :file:`samples/bluetooth/peripheral_hids_mouse`
 
-.. include:: /includes/build_and_run_ns.txt
+.. _peripheral_hids_mouse_ncd_build:
 
-.. include:: /includes/nRF54H20_erase_UICR.txt
+nRF Connect for Desktop build
+=============================
+
+To build this sample in the configuration variant that is compatible with `nRF Connect for Desktop`_, disable the following Bluetooth features:
```
Comment (peknis) [inline samples/bluetooth/peripheral_hids_mouse/README.rst:130]:
```suggestion
To build this sample in the configuration variant that is compatible with `Bluetooth Low Energy app`_, disable the following Bluetooth features:
```

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 7]
```diff
@@ -107,14 +121,32 @@ To build this sample with the :ref:`ble_rpc` library, add the following paramete
...
+
+* Privacy (:kconfig:option:`CONFIG_BT_PRIVACY`) - `nRF Connect for Desktop`_ does not fully support the Bluetooth Privacy feature by disallowing distribution of the Identity Resolving Key (IRK) during the pairing procedure.
+* High-duty directed advertising (:kconfig:option:`CONFIG_BT_DIRECTED_ADVERTISING`) - high-duty directed advertising with 3.75 ms advertising interval and 1.28 s duration prevents the subsequent undirected advertising from being reported in the scanning list of `nRF Connect for Desktop`_ .
```
Comment (peknis) [inline samples/bluetooth/peripheral_hids_mouse/README.rst:133]:
```suggestion
* High-duty directed advertising (:kconfig:option:`CONFIG_BT_DIRECTED_ADVERTISING`) - high-duty directed advertising with 3.75 ms advertising interval and 1.28 s duration prevents the subsequent undirected advertising from being reported in the scanning list of `Bluetooth Low Energy app`_ .
```

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 8]
```diff
@@ -161,6 +193,10 @@ Testing with nRF Connect for Desktop
 
 To test with `nRF Connect for Desktop`_, complete the following steps:
 
+.. note::
+   To execute the testing steps below, you must build the sample in the configuration that is compatible with `nRF Connect for Desktop`_.
```
Comment (peknis) [inline samples/bluetooth/peripheral_hids_mouse/README.rst:197]:
```suggestion
   To execute the testing steps below, you must build the sample in the configuration that is compatible with `Bluetooth Low Energy app`_.
```

File: samples/bluetooth/peripheral_hids_mouse/README.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (annwoj) [reply samples/bluetooth/peripheral_hids_mouse/README.rst:107]: @FrancescoSer is it okay to remove it in this PR or do you have a separate cleanup task planned?

[the remark this answers is not in the document]
Comment (FrancescoSer) [reply samples/bluetooth/peripheral_hids_mouse/README.rst:107]: This note should stay there, please check the note text contain in the file, it was updated do reflect 2.9 reality

**Review by peknis — APPROVED** [review]

**Comment by kapi-no** [comment]
Rebased to reflect the new doc codeowners changes

**Review by wiba-nordic — APPROVED** [review]

**Pull request merged**